### DISCUSSION TITLE
How to show the navbar in the docs page for all screen size?

### DISCUSSION THREAD

**Discussion by jpmti2016** [fuma-nama/fumadocs#1195] [Q&A]
![Screenshot from 2024-12-28 05-34-03](https://github.com/user-attachments/assets/de9825b3-2351-4b90-a863-86dcf74d8a2e)
![Screenshot from 2024-12-28 05-34-34](https://github.com/user-attachments/assets/73552e0c-4d81-4380-b89c-e6127f1bd71d)

Do I have to add a prop to DocsLayout or change some css var to  show the navbar on all screen sizes?

Otherwise if I share the main HomeLayout  how to disable the navbar in DocsLayout, or do I have to use only one layout type to avoid duplicate navbar on certain screen sizes.

**Comment by fuma-nama** [comment]
Use Notebook layout instead

https://fumadocs.vercel.app/docs/ui/layouts/docs#notebook
